cloud/lib/authentication/cognitoUserPoolCdkConstruct.ts

```diff
@@ -50,13 +61,56 @@ export class CognitoUserPoolCdkConstruct extends Construct {
         disableOAuth: false,
         supportedIdentityProviders: [
           cognito.UserPoolClientIdentityProvider.COGNITO,
+          //Supported providers generated depending on selected auth methods
+          cognito.UserPoolClientIdentityProvider.FACEBOOK,
+          cognito.UserPoolClientIdentityProvider.GOOGLE,
+          cognito.UserPoolClientIdentityProvider.APPLE,
         ],
+        oAuth: {
+          callbackUrls: [
+            process.env.REACT_APP_USER_POOL_REDIRECT_SIGN_IN ||
```

Comment:
Instead of setting default env variable here, can we add some file with env variables or update script strictly for development?

Reply:
Callback and logout urls are now stored in `.env.default` file which can be overwritten by `.env` file